### PR TITLE
Cordova support for Android ChildBrowser

### PR DESCRIPTION
**Pull request by alculquicondor**
I changed the imports from `com.phonegap` to `org.apache.cordova` and also casted `ctx` object to `Context`, so ChildBrowser plugin now works with cordova-1.5.0.jar

### CODE REVIEW

**Comment by RandyMcMillan** [comment]
Did you document your changes so devs can roll back if they need to?

File: Android/ChildBrowser/src/com/phonegap/plugins/childBrowser/ChildBrowser.java [hunk 1]
```diff
@@ -119,7 +120,7 @@ public String openExternal(String url, boolean usePhoneGap) {
         try {
             Intent intent = null;
             if (usePhoneGap) {
-                intent = new Intent().setClass(this.ctx, com.phonegap.DroidGap.class);
+                intent = new Intent().setClass((Context) this.ctx, org.apache.cordova.DroidGap.class);
```
Comment (alculquicondor) [inline Android/ChildBrowser/src/com/phonegap/plugins/childBrowser/ChildBrowser.java:123]: DroidGap class is now in `org.apache.cordova` package.
`setClass` method in `Intent` requires `(Context, Class<?>)` so casting from `CordovaInterface` to `Context` is needed.

File: Android/ChildBrowser/src/com/phonegap/plugins/childBrowser/ChildBrowser.java [hunk 2]
```diff
@@ -211,7 +212,7 @@ public String showWebPage(final String url, JSONObject options) {
         // Create dialog in new thread 
         Runnable runnable = new Runnable() {
             public void run() {
-                dialog = new Dialog(ctx);
+                dialog = new Dialog((Context) ctx);
```
Comment (alculquicondor) [inline Android/ChildBrowser/src/com/phonegap/plugins/childBrowser/ChildBrowser.java:215]: cast needed.

File: Android/ChildBrowser/src/com/phonegap/plugins/childBrowser/ChildBrowser.java [hunk 3]
```diff
@@ -357,7 +358,7 @@ private void sendUpdate(JSONObject obj, boolean keepCallback) {
      * The webview client receives notifications about appView
      */
     public class ChildBrowserClient extends WebViewClient {
-        PhonegapActivity ctx;
+        CordovaInterface ctx;
```
Comment (alculquicondor) [inline Android/ChildBrowser/src/com/phonegap/plugins/childBrowser/ChildBrowser.java:361]: `CordovaInterface` replaced `PhonegapActivity`.

**Comment by alculquicondor** [comment]
I hope that's enough. The changes weren't dramatical.

**Pull request merged**